README.md

```diff
@@ -59,7 +59,7 @@ The group ID is basically the metaID the lootbags will have; So if the lootbag h
 that will spawn items for that group will have the full ID 123:5. That's why you can't set a groupID below 1, or above 32767.
 A warning: Once you've defined a group ID, never change it. While lootbags won't cause corrupted worlds or crashes, recorded itemdrops are lost, as the GroupID is
 used to keep track of a players progress.
-Important: The GroupName, which is used to display the lootbag-name for players, must be edited in the file directly! That is because lazyness
+As of FILL_IN_VERSION, you can localize group names of lootbags. For example, for `GroupName="Legendary - Unique"`, the localization key will be `enhancedlootbags.group.legendary_-_unique`. Provide your own lang file and load it with mods like TXLoader. If it's not found, the raw GroupName is displayed.
```

Comment:
Requiring string concatenation for lang keys has a few issues, and I'm not sure we want to require other mods/modpacks to overwrite the enhancedlootbags namespace.